content/articles/2018-07-ggplot2-3-0-0.Rmarkdown

```diff
@@ -0,0 +1,173 @@
+To support quasiquotation in facetting we've added a new helper that works 
+similarly to `aes()`: `vars()`, short for variables, and instead of 
+`facet_grid(x + y ~ a + b)` you can now write `facet_grid(vars(x, y), vars(a, b))`. 
+The formula interface won't go away; but the new `vars()` interface should be 
+much easier to program with.
+
+```{r label-both}
+x_var <- quo(wt)
+y_var <- quo(mpg)
+group_var <- quo(cyl)
+
+ggplot(mtcars, aes(!!x_var, !!y_var)) + 
+  geom_point() + 
+  facet_wrap(vars(!!group_var))
+```
+
+## New features
+
+Below we describe just some of the new features and functionality added. For a full account of these improvements, please see [NEWS](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md).
+
+### sf
+
+Thanks to the help of Edzer Pebesma and the r-spatial team, ggplot2 now has full support for simple features through 
+[sf](https://r-spatial.github.io/sf/) using `geom_sf()` and `coord_sf()`; it now 
+automatically aligns CRS across layers, sets up the correct aspect ratio, and 
+draws a graticule.
+
+```{r sf, message = FALSE}
+nc <- sf::st_read(system.file("shape/nc.shp", package = "sf"))
+ggplot(nc) +
+  geom_sf() +
+  annotate("point", x = -80, y = 35, colour = "red", size = 4)
+```
+
+### Calculated aesthetics
+
+The new `stat()` function offers a cleaner, and better-documented syntax for 
+calculated-aesthetic variables. This replaces the older approach of surrounding 
+the variable name with `...`. Instead of using `aes(y = ..count..)`, you can use 
+`aes(y = stat(count))`. 
+
+This is particularly nice for more complex calculations, as `stat()` only needs 
+to be specified once; e.g. `aes(y = stat(count / max(count)))` rather than 
+`aes(y = ..count.. / max(..count..))`.
+
+### Tag
+
+In addition to title, subtitle, and caption, a new tag label has been added, for identifying plots. Add a tag with `labs(tag = "A")`, style it with the `plot.tag` theme element, and control position with the `plot.tag.position` theme.
+
+```{r tag}
+ggplot(mtcars) +
+  geom_point(aes(disp, mpg)) + 
+  labs(tag = 'A', title = 'Title of this plot')
+```
+
+### Layers: geoms, stats, and position adjustments
+
+The new `position_dodge2()` allows you to arrange the horizontal position of plots with variable widths (i.e. bars and rectangles, in addition to box plots). By default, `position_dodge2()` preserves the width of each element.
+
+```{r dodge2-single}
+ggplot(mtcars, aes(factor(cyl), fill = factor(vs))) +
+  geom_bar(position = position_dodge2(preserve = "single"))
+```
+
+You can choose to preserve the total width by setting the `preserve` argument to `"total"`. Thank you to Kara Woo for all of her work on this.
+
+```{r dodge2-total}
+ggplot(mtcars, aes(factor(cyl), fill = factor(vs))) +
+  geom_bar(position = position_dodge2(preserve = "total"))
+```
+
+### Scales and guides
+
+The new `scale_alpha()` function can be used to but can be used to visually down-weight less important observations.
```

Comment:
I think the news is that alpha now works for date-time variables, which isn't super exciting, so I think we can just drop this section.